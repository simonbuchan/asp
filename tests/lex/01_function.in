function functionName(argName:ArgType):ResultType {
    body;
}
