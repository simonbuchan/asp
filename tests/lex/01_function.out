1: keyword 'function'
1: id 'functionName'
1: op lparen '('
1: id 'argName'
1: op colon ':'
1: id 'ArgType'
1: op rparen ')'
1: op colon ':'
1: id 'ResultType'
1: op lcurly '{'
2: id 'body'
2: op semicolon ';'
3: op rcurly '}'
